src/implementations/react/components/button/stories/Index.jsx

```diff
@@ -6,9 +6,13 @@ import demoContentSecondary from '@ecl/ec-specs-button/demo/data--secondary';
 import demoContentCall from '@ecl/ec-specs-button/demo/data--call';
 import demoContentGhost from '@ecl/ec-specs-button/demo/data--ghost';
 import demoContentSearch from '@ecl/ec-specs-button/demo/data--search';
+import euDemoContentPrimary from '@ecl/ec-specs-button/demo/eu-data--primary';
```

Comment:
Relying on systems separation in terms of file systems. It's the same as what we have in v2.

As mentioned here https://github.com/ec-europa/europa-component-library/pull/1690/files#r496654688 I think the example demonstrating data structure difference is not fully matching foreseen hardship in input data. At the moment, there are no differences in primary button data and there is no need for any split.

On the other hand, if there were a difference in the input data for the primary button between ec and eu, it shouldn't be handled in storybook in my opinion. Rather, it should be part of the checks inside the templates. (which is currently react, but will be twig)

I don't remember where was the document by Antonio regarding how the platform plans to load assets based on system context, but i doubt it's a layer before templates.

The topic of input data is separate from the number of storybook instances we can have in the repository.

Reply:
I wanted simply to handle different content (not only, but mainly) in the demos we make based on the system in use, which is a use case that we already have and we would not be able to reproduce in v3 without a solution for it, even easy as setting a variable is, i'm not trying to guess anything that i cannot imagine... :)

The twig template is already the same for the two systems in v2, core-team is already using the same template in the two systems, so in ECL v2 we are already handling and at the moment without informing the templates about the context, they are "system agnostic", as i mentioned before this is handled with "no additional logic in the templates" and i wouldn't change this unless there is a clear need for that, but also that is a possibility for V3, not excluding that.

Then we have the track about potential, "overall" changes in the definition of the data structure to facilitate the integration, if there is anything we can do for that, but that's on a different level, hope we can work on that even before making any "real" component here...

Reply:
>>The twig template is already the same for the two systems in v2, core-team is already using the same template in the two systems, so in ECL v2 we are already handling and at the moment without informing the templates about the context, they are "system agnostic"

Ok. given template layer is solved, the demo here is showing a preparatory layer before templates selecting the right data based on a context of some sort. As I mentioned, if it's not breaking the way EWP is handling the data too, I don't mind having this selection logic on file-level in storybook